dags/direct2parquet_bigquery_load.py

```diff
@@ -110,22 +111,32 @@
 
     # Daily and last seen views on top of glean pings.
 
-    glean_clients_daily = bigquery_etl_query(
-        task_id='glean_clients_daily',
-        destination_table='glean_clients_daily_v1',
-        dataset_id='telemetry',
-        start_date=datetime(2019, 7, 1),
+    wait_for_copy_deduplicate = ExternalTaskSensor(
+        task_id="wait_for_copy_deduplicate",
+        external_dag_id="copy_deduplicate",
+        external_task_id="copy_deduplicate_all",
+        execution_delta=timedelta(hours=-1),
```

Comment:
this should be positive or it won't be pointing at the correct `copy_deduplicate` job, because it's schedule is one hour before this task, and [per the docs](https://airflow.readthedocs.io/en/stable/_modules/airflow/sensors/external_task_sensor.html): `For yesterday, use [positive!] datetime.timedelta(days=1)`
```suggestion
        execution_delta=timedelta(hours=1),
```

Reply:
Do I need an execution delta at all? Reading the docs, it makes it sound like it will default to the job executed on the same day, and makes no mention of the specific timestamp needing to match up.

Reply:
it's consistent with other implementations to have one, but I honestly have no clue if it's actually necessary.

Reply:
also, it may only be necessary for same-day deltas where the dependency starts *after* the sensor